doc/Building_FRR_on_Ubuntu1604.md

```diff
@@ -0,0 +1,111 @@
+Building FRR on Ubuntu 12.04LTS from Git Source
+===============================================
+
+- MPLS is not supported on `Ubuntu 12.04` with default kernel. MPLS requires 
```

Comment:
So this "how to upgrade" is similar to the link I've added where people can download
a newer kernel. So I assume this should cover it.